mcproto/player.py

```diff
@@ -118,6 +119,16 @@ def op(self) -> None:
     def deop(self) -> None:
         HasStub.runCommand(self, f"deop {self.name}")
 
+    def show_title(self, text: str, typ: Literal["actionbar", "subtitle", "title"] = "title", color: COLOR = "gray", bold: bool = "false", italic: bool = "false", strikethrough: bool = "false", underlined: bool = "false", obfuscated: bool = "false", duration: int = 1, fade_in: int = 5, fade_out: int = 1) -> None:
+        HasStub.runCommand(self, f'title {self.name} times {fade_in}s {duration}s {fade_out}s')
+        HasStub.runCommand(self, f'title {self.name} {typ} ' + '{' + f'"text":"{text}","color":"{color}","bold":{bold},"italic":{italic},"strikethrough":{strikethrough},"underlined":{underlined},"obfuscated":{obfuscated}' + '}')
```

Comment:
Da wir in der Klasse Player > Entity hier sind, können wir ruhig die überschriebene "runCommand" Funktion dieser Klasse verwenden. Hier direkt HasStub.runCommand zu verwenden umgeht die Klassenhierachie.
Wenn man das macht, könnte man auch bei allen Befehlen einfach 'title @s ...' schreiben, da der Player identifier schon in Entity.runCommand gesetzt wird :+1:

Reply:
Bitte diesen Punkt noch ausbessern, alle Instanzen von `HasStub.runCommand` durch `self.runCommand` ersetzten, und im Falle von der Player-Klasse, mit `title @s ...` anstelle von `title {self.name} ...`